cli/ballerina-cli/src/test/resources/test-resources/command-outputs/windows/build-tool-with-diagnostics.txt

```diff
@@ -1 +1,8 @@
+
+Executing Build Tools
+
+	openapi(generate-delivery-client)
+
 ERROR [openAPI sample build tool:(1:1,1:1)] The provided filePath does not exist
```

Comment:
Does `(1:1,1:1)` mean the location is unknown?

Reply:
This is caused by using Null Locations with the `DefaultDiagnostic` type. Will change the implementation to wrap this using `PackageDiagnostics`.